README.md

```diff
@@ -28,6 +28,14 @@ Before running the demo app, you need to download and install the project depend
 npm install
 ```
 
+The Gutenberg packages need to be installed and built. This is done via the following commands:
+
+```
+pushd gutenberg
```

Comment:
Should we put it in the shell script and run as `postinstall` script for the repository?

Reply:
Perhaps. I refrained from automating it from the beginning so to give us time/space to see how we use it actually but, I can give it a try now and see.

Reply:
So, gave `postinstall`. Our patterns these days involve more than a few `yarn install` attempts and that will cause `npm i` to be called too many times, unnecessarily. Ideally, we want `npm i` to be called rarely, like, when the GB repo is cloned or when the GB branch is changed.

I think we can leave `npm i` out of automation for the following days/weeks and re-assess, depending on the patterns we notice.

Reply:
And btw, thanks for the review @gziolo !